example/matrixAddWithMdspan/src/matrixAddMdSpan.cpp

```diff
@@ -0,0 +1,182 @@
+/* Copyright 2024 Mehmet Yusufoglu, Simeon Ehrig, Andrea Bocci
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+#include <alpaka/alpaka.hpp>
+// Needed for running example for all backends available; one by one
+#include <alpaka/example/ExecuteForEachAccTag.hpp>
+
+#include <experimental/mdspan>
+#include <iostream>
+
+//! Matrix addition example by using mdspan data structure
+
+//! Some simple type traits for checking the types
+//! isMdspan simply checks if a type is of type std::experimental::mdspan or not
+//! Primary template for is_mdspan (defaults to false)
+template<typename T>
+struct isMdspan : std::false_type
+{
+};
+
+//! Specialization for mdspan with four template arguments
+template<typename ElementType, typename Extents, typename LayoutPolicy, typename AccessorPolicy>
+struct isMdspan<std::experimental::mdspan<ElementType, Extents, LayoutPolicy, AccessorPolicy>> : std::true_type
```

Comment:
can you rename this to `IsMdspan` ?

and add
```c++
template <typename T>
inline constexpr bool is_mdspan = IsMdspan<T>::value;
```
?

Reply:
done,  thanks.